src/test/groovy/io/codearte/jfairy/producer/person/locale/de/DePassportNumberProviderSpec.groovy

```diff
@@ -0,0 +1,36 @@
+package io.codearte.jfairy.producer.person.locale.de
+
+import io.codearte.jfairy.producer.BaseProducer
+import spock.lang.Specification
+import spock.lang.Unroll
+
+/**
+ * @author Roland Weisleder
+ */
+class DePassportNumberProviderSpec extends Specification {
+
+	private BaseProducer baseProducer = new BaseProducer(new Random())
+	private DePassportNumberProvider generator = new DePassportNumberProvider(baseProducer)
+
+	@Unroll
+	def "Should validate #passportNumber as #valid"() {
+
+		expect:
+			generator.isValid(passportNumber) == valid
+
+		where:
+			passportNumber | valid
+			"C22000129"    | true
```

Comment:
Please convert to Groovy Strings in place of GStrings.
Please remove unnecessary empty lines in test methods.